lib/transformer.dart

```diff
@@ -98,6 +98,26 @@ class WebSkinDartTransformer extends Transformer implements LazyTransformer {
     TransformedSourceFile transformedFile = new TransformedSourceFile(sourceFile);
     TransformLogger logger = new JetBrainsFriendlyLogger(transform.logger);
 
+    var ignoreCommentPattern = new RegExp(r'\/\/\s?ignore:\s?uri_does_not_exist,\s?uri_has_not_been_generated');
+    var partPattern = new RegExp(r'(\s*?)part\s(.*?).overReact.g.dart(.;)');
```

Comment:
Not sure if this was already considered, but another option would be to emit an empty `.g.dart` file from the transformer, which is probably more robust than trying to do this transformation.

Just putting it out there.